### ISSUE TITLE
Enhancement: Improve Toggle Button Style 

### ISSUE THREAD

**Issue by kspilarski** [Automattic/wp-calypso#79154]
### What

Improve the style of the toggle buttons by adding a check mark or making the colors easier to see which options or settings are enabled and not. 

### Why

The toggle button is sometimes confusing for users whether an option or a setting is currently enabled or not. Also depending on the Dashboard Color Scheme selected, some toggle buttons can obviously show as enabled, while some like _Contrast_ and _Nightfall_ do not. 

### How

In the Interface settings in https://wordpress.com/me/account  Dashboard color scheme, some colors like  _Contrast_ and _Nightfall_ show the active and inactive toggle buttons as black and white making it confusing whether it's enabled or not. 

### Dashboard color scheme: _Contrast_ 

<img width="878" alt="Contrast - discussion settings" src="https://github.com/Automattic/wp-calypso/assets/36863633/6eb2ac4f-d5df-4526-96fa-fd0e37e5c31b">

### Dashboard color scheme: _Nightfall_ 

<img width="854" alt="Screenshot 2023-07-08 at 20 27 47" src="https://github.com/Automattic/wp-calypso/assets/36863633/45eca341-0967-4bde-88fa-115375d411d8">

<hr /> 

### Other examples

Dashboard color scheme: _Default_  

<img width="787" alt="Default - discussion settings" src="https://github.com/Automattic/wp-calypso/assets/36863633/59c17535-643e-4f5e-b2d7-5dff4b161a2d">

The Privacy settings with the radio and check mark buttons make it easy to spot which one is selected: 

<img width="791" alt="Screenshot 2023-07-01 at 12 01 02" src="https://github.com/Automattic/wp-calypso/assets/36863633/02e24c56-0ae5-4cab-ac18-15a9573dad6b">

The Security settings have check marks that makes it easy to see which ones are done and exclamation points for those that are not selected: 

<img width="670" alt="Security settings - checkmark" src="https://github.com/Automattic/wp-calypso/assets/36863633/1c32322d-13bb-416c-ab2a-0c92fbc5b003">

**Comment by cuemarie** [comment]
📌 ACTIONS
- Feature request kept
